api/share/utils.py

```diff
@@ -241,7 +241,6 @@ def format_registration(registration, *args, **kwargs):
             'description': registration.description or '',
             'is_deleted': not registration.is_public or registration.is_deleted or is_qa,
             'date_published': registration.registered_date.isoformat() if registration.registered_date else None,
-            'registration_type': registration.registered_schema.first().name if registration.registered_schema.exists() else None,
```

Comment:
So the reason this was failing on the OSF provider and not the others is because the OSF provider was set to a `share_publish_type` of `preprint` rather than `registration`, so these fields weren't valid. So could you put this line back in, and also put `'withdrawn': registration.is_retracted,` back in under it, so we have that info being indexed in SHARE again?

Reply:
@brianjgeiger yep I'll prioritize that.